docs/resources/Guild.md

```diff
@@ -130,6 +130,7 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 
 | Feature                          | Description                                                                                                         |
 | -------------------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| ANIMATED_BANNER                  | guild has access to set an animated guild banner image                                                              |
```

Comment:
is "image" necessary? since it can be a gif, also to be standardized with the one below (`ANIMATED_ICON`)